iati/core/schemas.py

```diff
@@ -11,8 +11,8 @@ class Schema(object):
     """Represenation of a Schema as defined within the IATI SSOT.
 
     Attributes:
-        name (str): The name of the Schema.
         codelists (set): The Codelists asspciated with this Schema. This is a read-only attribute.
+        root_element_name (str): The name of the root element within the schema - i.e. 'iati-activities' for the activity schema and 'iati-organisations' for the organisation schema.
```

Comment:
Slightly questionable having documentation for child classes within the parent class, but seems OK in this case.

How do the generated docs look?